UI/window-basic-settings.hpp

```diff
@@ -383,6 +383,9 @@ private slots:
 	bool ServiceAndACodecCompatible();
 	bool ServiceSupportsCodecCheck();
 
+	inline bool allowsMultiTrack(const char *protocol);
```

Comment:
Drop inline to make compilers stop complaining
```suggestion
	bool allowsMultiTrack(const char *protocol);
```

Reply:
Can be put as resolved

Reply:
moved the function to window-basic-settings-stream.cpp ; macos and flatpak no longer complain.